src/FSharpLint.Console/FSharpLint.Console.fsproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
Is this project file supposed to have both ```TargetFramework``` and ```TargetFrameworks```

Reply:
Good catch, @webwarrior-ws please fix

Reply:
fixed